### ISSUE TITLE
Allow compiling without clipboard support

### ISSUE THREAD

**Issue by karx1** [magic-wormhole/magic-wormhole.rs#157]
Hi, first of all I just wanted to say that this is a great project, and I use it almost daily.

This project won't compile on systems without a graphical display, such as Termux on Android and also servers which don't have X11 installed. It would be great if you could add a feature flag to disable the clipboard support so I can compile and use this tool on those systems. I don't want to just use the python version.

Thanks in advance,
karx

**Comment by piegamesde** [comment]
Yes, this should be doable. However, I shall note that the dependency is on libxcb only and not on X11 itself. This is a key difference, since the former is a rather small package that shouldn't hurt too much in a headless installation.

I've already had this topic come up with a packager, and the optimal solution would be to have libxcb as an optional, run-time dependency. But I don't really know how to do this in Rust, so a feature flag might be the best available option.

**Comment by pepa65** [comment]
This would be a great feature to have, to compile without needing libxcb and not having automatic clipboard usage, usable on headless machines.
